lightstrip.py

```diff
@@ -47,29 +47,29 @@ def assign_mode(self, current_mode: str, last_mode: str):
 
         elif current_mode == modes["READY_TO_SHOOT"]:
             # Ready to shoot, railgun pattern (requested by drivers)
-            self.primary = (0, 255, 0)
+            self.primary = (0, 255, 0) # Green
```

Comment:
⭐ An even better way to do this would be to create variables for these above like:
```python
GREEN = (0, 255, 0)
```

And then in this code you can just have:
```python
self.primary = GREEN
```